plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1692,6 +1694,582 @@ public void testMultiIndexAlias()
+                .put("elasticsearch.host", elasticsearch.getAddress().getHost())
+                .put("elasticsearch.port", Integer.toString(elasticsearch.getAddress().getPort()))
+                // Node discovery relies on the publish_address exposed via the Elasticseach API
+                // This doesn't work well within a docker environment that maps ES's port to a random public port
+                .put("elasticsearch.ignore-publish-address", "true")
+                .put("elasticsearch.default-schema-name", "tpch")
+                .put("elasticsearch.scroll-size", "1000")
+                .put("elasticsearch.scroll-timeout", "1m")
+                .put("elasticsearch.request-timeout", "2m")
+                .put("elasticsearch.union-schema-indices-for-alias", "true")
+                .put("elasticsearch.fail-on-alias-schema-mismatch", "false")
+                .build();
+
+        String catalogName = format("elasticsearch_%s", randomTableSuffix());
+
+        getQueryRunner().createCatalog(catalogName, "elasticsearch", config);
+
+        String indexName1 = format("index_%s", randomTableSuffix());
+        @Language("JSON")
+        String properties = "" +
+                "{" +
+                "  \"properties\":{" +
+                "    \"name\":   { \"type\": \"text\" }," +
+                "    \"description\":   { \"type\": \"text\" }," +
+                "    \"rackid\":   { \"type\": \"keyword\" }" +
+                "  }" +
+                "}";
+        createIndex(indexName1, properties);
+
+        String indexName2 = format("index_%s", randomTableSuffix());
+        properties = "" +
+                "{" +
+                "  \"properties\":{" +
+                "    \"name\":   { \"type\": \"integer\" }," +
+                "    \"description\":   {" +
+                "       \"properties\": {" +
+                "           \"value\": {\"type\": \"text\" }" +
+                "       }" +
+                "    }," +
+                "    \"bookid\":   { \"type\": \"long\" }" +
+                "  }" +
+                "}";
+        createIndex(indexName2, properties);
+
+        String aliasName = format("alias_%s", randomTableSuffix());
+
+        addAlias(indexName1, aliasName);
+        addAlias(indexName2, aliasName);
+
+        Session session = testSessionBuilder().setCatalog(catalogName).setSchema("tpch").build();
+        MaterializedResult expectedColumns = resultBuilder(session, VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("description", "varchar", "", "")
+                .row("bookid", "bigint", "", "")
+                .row("name", "varchar", "", "")
+                .row("rackid", "varchar", "", "")
+                .build();
+        MaterializedResult actualColumns = computeActual(session, "DESCRIBE " + aliasName);
+        assertEquals(actualColumns, expectedColumns);
+        deleteIndex(indexName1);
+        deleteIndex(indexName2);
```

Comment:
can we also have tests with isArray, asRawJson set and with datetime, scaledfloat type?

Reply:
Sure. Will add them.